### ISSUE TITLE
The 'a0' in LiftDragPlugin does not obey the aerodynamics habbits.

### ISSUE THREAD

**Issue by petertheprocess** [osrf/gazebo#2831]
https://github.com/osrf/gazebo/blob/a9f093f2fc7fa7fc49efa73719f6536ad28f8af7/plugins/LiftDragPlugin.cc#L254-L255
https://github.com/osrf/gazebo/blob/a9f093f2fc7fa7fc49efa73719f6536ad28f8af7/plugins/LiftDragPlugin.cc#L272-L274
 For people who know aerodynamics, 'a0' can be easily misunderstood as 'Zero-Lift AOA' which is just the opposite number of present definition.
So I suggest changing the code like this.
```
this->alpha =  acos(cosAlpha) - this->alpha0; 
```
https://github.com/PX4/sitl_gazebo/issues/586#issue-684509105

**Comment by chapulina** [comment]
I don't think this is likely to change, since it would break existing code that is accommodating the current math. I'd suggest you open a PR updating the tutorial with clarifications for users:

http://gazebosim.org/tutorials?tut=aerodynamics&cat=physics#Inputparameters

The same math was [ported to Ignition](https://github.com/ignitionrobotics/ign-gazebo/blob/15c56c6e99e64b1a43ed9e79b30e6d2257e227b5/src/systems/lift_drag/LiftDrag.cc#L314-L315). The math could be updated there on future Ignition versions.
